hive3/src/main/java/org/apache/hadoop/hive/ql/io/orc/OrcSplit.java

```diff
@@ -191,12 +191,12 @@ public boolean hasFooter() {
   }
 
   /**
-   * @return {@code true} if file schema doesn't have Acid metadata columns
    * Such file may be in a delta_x_y/ or base_x due to being added via
    * "load data" command.  It could be at partition|table root due to table having
    * been converted from non-acid to acid table.  It could even be something like
    * "warehouse/t/HIVE_UNION_SUBDIR_15/000000_0" if it was written by an
    * "insert into t select ... from A union all select ... from B"
+   * @return {@code true} if file schema doesn't have Acid metadata columns
```

Comment:
I think this can be improved, but moving this line to the end doesn't help readability. Can you rewrite this doc instead?

Reply:
removing redundant e.g.

Reply:
I think the right fix is to document the method, not to just move the `@return`. Can you fix the documentation here instead? Javadoc should start with a short (one sentence) description and then go into more detail if needed after a paragraph break.

Reply:
thanks, I got it and tried to description it.